src/styles/_variables.scss

```diff
@@ -46,3 +49,36 @@ $bigger-nice-index: 70;
 $nice-index: 69;
 $almost-nice-index: 68;
 $not-nice-index: 67;
+
+//themes
```

Comment:
move to another file, change variables names like scss names, put CSS variables into scss vars